src/main/java/core/basesyntax/service/activity/BalanceActivityService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.activity;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.FruitServiceImpl;
+import core.basesyntax.service.GetListOfActivity;
+import core.basesyntax.service.GetListOfActivityImpl;
+import java.util.List;
+
+public class BalanceActivityService implements ActivityService {
+    private static final int FRUIT_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
```

Comment:
you have such values in several places, let's split in one place and pass already parsed values in activity method to eliminate duplication